backend/src/constants/groups.ts

```diff
@@ -1,3 +1,10 @@
-// this file is duplicated in `backend/src/constants/group` and `webapp/constants/group.js`
-export const DESCRIPTION_WITHOUT_HTML_LENGTH_MIN = 3 // with removed HTML tags
-export const DESCRIPTION_EXCERPT_HTML_LENGTH = 250 // with removed HTML tags
+// configurable values see: 'backend/src/constants/groupsDefault', 'webapp/constants/groupsDefault'
+// configurable values see: 'backend/src/constants/groups', 'webapp/constants/groups'
+
+import * as groupsDefault from './groupsDefault'
+import * as groupsBranding from '../branding/groupsBranding'
+
+export const { DESCRIPTION_WITHOUT_HTML_LENGTH_MIN, DESCRIPTION_EXCERPT_HTML_LENGTH } = {
+  ...groupsDefault,
+  ...groupsBranding,
+}
```

Comment:
Why export specific variables? Is there a typescript advantage doing this?

```suggestion
export default {
  ...groupsDefault,
  ...groupsBranding,
}
```

Reply:
I did it this way because this export is imported as:

```ts
import { DESCRIPTION_WITHOUT_HTML_LENGTH_MIN } from '../../constants/groups'
```

So you suggestion creates an error:

```bash
src/middleware/excerptMiddleware.ts:2:10 - error TS2614: Module '"../constants/groups"' has no exported member 'DESCRIPTION_EXCERPT_HTML_LENGTH'. Did you mean to use 'import DESCRIPTION_EXCERPT_HTML_LENGTH from "../constants/groups"' instead?
```

It's not imported as whole object where a property is selected.

The way I do it has the following good sides:

- if I go on refactoring I have not to rewrite all the imports
- it's clear which configs can be passed through and all others are filtered out